source/type/pb_converter.h

```diff
@@ -34,6 +34,85 @@ www.navitia.io
 #include "type/response.pb.h"
 #include "type/pt_data.h"
 
+namespace pt = boost::posix_time;
+
+// we don't use a boolean to have a more type checks
+enum class DumpMessage {
+    Yes,
+    No
+};
+
+#define null_time_period boost::posix_time::time_period(boost::posix_time::not_a_date_time, boost::posix_time::seconds(0))
```

Comment:
How about using
`static const auto null_time_period = boost::posix_time::time_period(boost::posix_time::not_a_date_time, boost::posix_time::seconds(0));` 
or 
`constexpr auto null_time_period = boost::posix_time::time_period(boost::posix_time::not_a_date_time, boost::posix_time::seconds(0));`?

with that, we can avoid using `#define` and `null_time_period` will be calculated at compile time.